xchange-poloniex/src/main/java/org/knowm/xchange/poloniex/PoloniexUtils.java

```diff
@@ -21,7 +21,7 @@
 
   public static String toPairString(CurrencyPair currencyPair) {
 
-    String pairString = currencyPair.counter.getCurrencyCode().toUpperCase() + "_" + currencyPair.base.getCurrencyCode().toUpperCase();
+    String pairString = currencyPair.base.getCurrencyCode().toUpperCase() + "_" + currencyPair.counter.getCurrencyCode().toUpperCase();
```

Comment:
@ikosyanenko 
@timmolter 

what is this change about???
its rolled back in the current version, but still... not necessary stuff